File: dayTwo/Python/book_suggestion_system.py
import random

library = ["Gift of men","King of persia","Lord of the rings","Super bowl"]
def add_book(bookToAdd):
    library.append(bookToAdd)
    if bookToAdd in library:
        return "Book added succesfully"
    return "Book not in library"

def remove_book(bookToRemove):
    library.remove(bookToRemove)
    if bookToRemove not in library:
        return "Book removed successfully"
    return "book not found"

def update_book(book_to_update,update):
    if book_to_update in library:
        position = library.index(book_to_update)
        library[position] = update
        return "Book updated successfully"    
    else:
        return "Book not found"                


def show_books():
    if not library:
        return "No books found"
    return library


def suggest_book():
    if len(library) > 0:
        first_rand = 1 + random.randint(0,100)
        second_rand = random.randint(0,len(library)-1)
        
        return f"{library[second_rand]} page {first_rand}"
    else:
        return "no book found"

File: dayTwo/javascript/bookSuggestionSystem.js
module.exports = {showBooks,addBook,removeBook,updateBook,getSuggestions};

let library = ["5 love languages","Gift of men","Followers of them","The last days","Philosophy of the wise","internet of things"];

//show books function
function showBooks(){
    return library;
}

//add book function
function addBook(bookToAdd){
    library.push(bookToAdd);
    if(library.includes(bookToAdd)){
        return "Book added successfully";
    }
    return "Book not added";
}


//removeBook
function removeBook(bookToRemove){
    if(library.includes(bookToRemove)){
        let index = library.indexOf(bookToRemove);
        library.splice(index,1);
        return "Book removed successfully";
    }
    return "Book not found";
}


//update book
function updateBook(bookToUpdate,update){
    if(library.includes(bookToUpdate)){
        let index = library.indexOf(bookToUpdate);
        library[index] = update;
        return "Book updated successfully";
    }
    return "book not found";    
}


function getSuggestions(){
    if(library.length > 0){
        let suggIndex = Math.floor(Math.random() * library.length);
        let pageRand = Math.round(Math.random() * 100);
        return library[suggIndex] + " page " + pageRand; 

    }
  
}







